test/js/ez-asynchronous-block.js

```diff
@@ -55,6 +55,22 @@ describe('ez-asynchronous-block', function() {
         });
     });
 
+    describe('redirection', function () {
+        it('should be redirected after 205 status', function (done) {
+            const response = new Response(null, {'status': 205});
+            const fetch = sinon.stub(window, 'fetch', function () {
+                return Promise.resolve(response);
+            });
+
+            element.addEventListener('ez:navigateTo', function () {
```

Comment:
to also test that the event is correctly configured to bubble, you should add the listener to the document, which also means you have to manually called `removeEventListener` to avoid side effects